src/libs/SidebarUtils.js

```diff
@@ -300,7 +300,13 @@ function getOptionData(reportID) {
     }
 
     if ((result.isChatRoom || result.isPolicyExpenseChat || result.isThread || result.isTaskReport) && !result.isArchivedRoom) {
-        result.alternateText = lastMessageTextFromReport.length > 0 ? lastMessageText : Localize.translate(preferredLocale, 'report.noActivityYet');
+        const lastAction = visibleReportActionItems[report.reportID];
+        if (lastAction && lastAction.actionName && lastAction.actionName === CONST.REPORT.ACTIONS.TYPE.RENAMED) {
```

Comment:
same as above can we use `lodashGet` to avoid multiple null checks?

Reply:
@Ollyws 
Actually you have done it in the below line itself, on this line we could change to 
```
lodashGet(lastAction, 'actionName', '') === CONST.REPORT.ACTIONS.TYPE.RENAMED
```
this way we can avoid multiple null checks and lodashGet will take care of it

Reply:
@Ollyws Do let me know if you require more information on this?